src/wiki/getting-started/installing-java.md

```diff
@@ -35,58 +80,71 @@ sudo dnf install java-1.8.0-openjdk java-17-openjdk
 ### Void Linux
 
 ```bash
-sudo xbps-install openjdk17-jre openjdk8-jre
+sudo xbps-install openjdk17 openjdk8
 ```
 
-### Arch Linux
+### Arch Linux, Manjaro, EndeavorOS, Garuda
 
 ```bash
-sudo pacman -S jre17-openjdk jre8-openjdk
+sudo pacman -S jdk17-openjdk jdk8-openjdk
 ```
 
-### Ubuntu
+### Ubuntu, Pop!\_OS, Linux Mint, Zorin OS, or elementaryOS
 
 ```bash
-sudo apt install openjdk-17-jre openjdk-8-jre
+sudo apt install openjdk-17-jdk openjdk-8-jdk
 ```
 
-### Debian
+### Debian, MX Linux
 
 ```bash
-sudo apt install openjdk-17-jre
+sudo apt install openjdk-17-jdk
 ```
 
-Java 8 is not available from Debian repos for unknown reasons, but you can [get a deb from Azul](https://www.azul.com/downloads/?version=java-8-lts&os=debian&package=jre&show-old-builds=true)
+Java 8 is not available in Debian 10+ due to lack of security support, but you can use the Adoptium repository for security support until 2026:
 
-### Flatpak
+```bash
```

Comment:
You can use Azul for this too

Reply:
Good idea, the Azul Debian repository actually supports more architectures. I think Temurin dropped 32-bit only because Ubuntu did.

Reply:
I don't even think anyone uses prism with Linux x86, you can only use up to Minecraft 1.12.2 with it because lwjgl